src/omnicore/rpc.cpp

```diff
@@ -848,67 +850,61 @@ int check_prop_valid(int64_t tmpPropId, string error, string exist_error ) {
 
 Value getorderbook_MP(const Array& params, bool fHelp)
 {
-   if (fHelp || params.size() < 1)
+   if (fHelp || params.size() < 1 || params.size() > 2)
         throw runtime_error(
             "getorderbook_MP property_id1 ( property_id2 )\n"
-            "\nAllows user to request active order information from the order book\n"
-            
+            "\nRequest active trade information from the MetaDEx\n"
+
             "\nArguments:\n"
-            "1. property_id1            (int, required) amount owned to up on sale\n"
-            "2. property_id2         (int, optional) property owned to put up on sale\n"
+            "1. propertyid           (int, required) filter orders on propertyid for sale\n"
+            "2. propertyid           (int, optional) filter orders on propertyid desired\n"
```

Comment:
Is "on" correct here, or should it be "filter by id"?

`int` should be changed to `number`, to mirror the types mentioned in the JSON spec.

Default values for optional arguments should be included, if applicable.

Result and examples should be added, similar to:
```
"\nResult:\n"
"{\n"
"  \"balance\" : \"x.xxxxxxxx\",   (string) the available balance of the address\n"
"  \"reserved\" : \"x.xxxxxxxx\"   (string) the amount reserved by sell offers and accepts\n"
"}\n"
"\nExamples:\n"
+ HelpExampleCli("getbalance_MP", "\"1EXoDusjGwvnjZUyKkxZ4UHEf77z6A5S4P\" 1")
+ HelpExampleRpc("getbalance_MP", "\"1EXoDusjGwvnjZUyKkxZ4UHEf77z6A5S4P\", 1")
```
Instead of a schema/example as result, a short description may be sufficient, such as "A JSON array of trade objects"

The examples may include actual numbers, or placeholders.

This is based on the discussion of https://github.com/OmniLayer/omnicore/issues/14#issuecomment-105420626.